java/pom.xml

```diff
@@ -390,6 +394,38 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-jdbc</artifactId>
```

Comment:
If a system has hive available, is this jar likely to already be on the classpath (this is common for other hadoop libraries)? If so, we usually set `<scope>provided</scope>` for these types of dependencies to keep the size of our jar down and not worry about dependency mismatches. You should be able to drop all the exclusions as well.

Reply:
This is true. Thanks for explaining why I should use `provided`